src/content/docs/tips/shadcn.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Setting up Shadcn UI
+---
+
+import { Tabs, TabItem, Aside, Steps,LinkCard } from "@astrojs/starlight/components";
+import ExampleLinks from '~/components/ExampleLinks.astro'
+
+## Examples
+
+If you want to see a more interactive example that might help explain or let you test things out better you can: 
+
+<ExampleLinks 
+  stackblitzLink="https://stackblitz.com/~/github.com/hkbertoson/astro-shadcn-ui"
+  githubLink="https://github.com/hkbertoson/astro-shadcn-ui"
+/>
+
+
+<Steps>
+1. Install react using your preferred package manager:
+    <Tabs syncKey="shadcn">
+      <TabItem label="npm" icon="seti:npm">
+        ```sh
+        npx astro add react
+        ```
+      </TabItem>
+      <TabItem label="pnpm" icon="pnpm">
+        ```sh
+        pnpx astro add react
+        ```
+      </TabItem>
+      <TabItem label="yarn" icon="seti:yarn">
+        ```sh
+        npx astro add react
+        ```
+      </TabItem>
+      <TabItem label="bun" icon="bun">
```

Comment:
Do we show bun in other examples? If not let's remove it

Reply:
Separately, we could use https://github.com/HiDeoo/starlight-package-managers